apps/dsp-app/src/styles/_elements.scss

```diff
@@ -363,18 +363,18 @@ a,
       position: absolute;
       left: 0;
     }
-
+    /* TODO(mdc-migration): The following rule targets internal classes of list that may no longer apply for the MDC version. */
```

Comment:
Delete the comment

Reply:
comment and unused css removed in [fbbc503](https://github.com/dasch-swiss/dsp-das/pull/1109/commits/fbbc503a50cd8e8cede7f19e53439694772c4533)